src/main/java/edu/usf/cutr/tba/manager/TravelBehaviorDataAnalysisManager.java

```diff
@@ -68,7 +72,10 @@ public void processData() {
         // create csv file and add the header
         mCSVFileWriter.createHeader(TravelBehaviorRecord.CSV_HEADER);
 
-        if (mProgramOptions.getUserId() == null) {
+        if (mProgramOptions.getMultiUserId() != null) {
+            // analyze all data for a specific list of userIds
+            analyzeListOfUserIdTravelBehaviorData(mProgramOptions.getMultiUserId());
+        } else if (mProgramOptions.getUserId() == null) {
```

Comment:
I would invert the last two parts of this IF statement so the two exports for subsets of users comes first, and the full export of all users comes last.

Reply:
Resolved in the lasts commits.